.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
![not gonna be part of your system!](https://user-images.githubusercontent.com/517302/46563632-dc971e00-c8bf-11e8-9dc7-0228fbe0e207.gif)

(👍)